aditional.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+type unexportedType string
+
+func ExportedFunc() unexportedType {
+	return unexportedType("some string")
```

Comment:
```suggestion
type ExportedType string

func ExportedFunc() ExportedType {
	return ExportedType("some string")
}
```
The type `unexportedType` should be exported to be accessible from other packages.  Exported names in Go start with an uppercase letter.